### PR TITLE
CTOR-1148 : Refresh network-versa-director-restapi

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh network-versa-director-restapi

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] 25.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-versa-director-restapi.md [hunk 1]
```diff
@@ -1,71 +1,109 @@
...
-<Tabs groupId="sync">
-<TabItem value="Hosts" label="Hosts">
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Net-Versa-Director-Device-Restapi-custom** est utilisé.
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias | Modèle de service                       | Description                  | Découverte |
+|:------|:----------------------------------------|:-----------------------------|:----------:|
+| Paths | Net-Versa-Director-Paths-Restapi-custom | Contrôle le statut des paths | X          |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-versa-director-restapi.md:39]:
```suggestion
| Paths | Net-Versa-Director-Paths-Restapi-custom | Contrôle le statut des chemins | X          |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-versa-director-restapi.md [hunk 2]
```diff
@@ -1,71 +1,109 @@
...
+| paths.down.count                | count |
+| *grp1*~paths.up.count           | count |
+| *grp1*~paths.down.count         | count |
+| *grp1*~*grp21*#paths.up.count   | count |
+| *grp1*~*grp22*#paths.up.count   | count |
+| *grp1*~*grp21*#paths.down.count | count |
+| *grp1*~*grp22*#paths.down.count | count |
 
 </TabItem>
 </Tabs>
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-versa-director-restapi.md:109]: Référez-vous, pas référencez-vous

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-versa-director-restapi.md [hunk 3]
```diff
@@ -81,165 +119,431 @@ Référencez-vous à la documentation officielle de Versa Networks pour l'activa
...
-  --critical-status='%{ping_status} ne "reachable" or %{services_status} ne "good"' \
-  --verbose
+/usr/lib/centreon/plugins/centreon_versa_director_restapi.pl \
+	--plugin network::versa::director::restapi::plugin \
+	--mode=paths \
+	--hostname='' \
+	--port='9182' \
+	--proto='https' \
+	--api-username='' \
+	--api-password='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-versa-director-restapi.md:371]: username and password

File: pp/integrations/plugin-packs/procedures/network-versa-director-restapi.md [hunk 4]
```diff
@@ -81,169 +118,431 @@ Please refer to the manufacturer documentation to achieve this.
...
-  --critical-status='%{ping_status} ne "reachable" or %{services_status} ne "good"' \
-  --verbose
+/usr/lib/centreon/plugins/centreon_versa_director_restapi.pl \
+	--plugin network::versa::director::restapi::plugin \
+	--mode=paths \
+	--hostname='' \
+	--port='9182' \
+	--proto='https' \
+	--api-username='' \
+	--api-password='' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-versa-director-restapi.md:370]: username and password

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-versa-director-restapi.md (hunk 3 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-versa-director-restapi.md:371]:
```suggestion
	--hostname='10.0.0.1' \
	--port='9182' \
	--proto='https' \
	--api-username='XXXX' \
	--api-password='XXXX' \
```

File: pp/integrations/plugin-packs/procedures/network-versa-director-restapi.md (hunk 4 above)
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-versa-director-restapi.md:370]:
```suggestion
	--hostname='10.0.0.1' \
	--port='9182' \
	--proto='https' \
	--api-username='XXXX' \
	--api-password='XXXX' \
```

**Review by cg-tw — DISMISSED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**